pyproject.toml

```
@@ -1,22 +1,74 @@
 [build-system]
 requires = ["setuptools>=75.6.0", "wheel", "setuptools_scm[toml]>=5.0"]
+build-backend = "setuptools.build_meta"
 
-[tool.mypy]
-exclude = "build/"
-plugins = ["pydantic.mypy"]
+[project]
+name = "evm-trace"
```

I like to have this here
```suggestion
name = "evm-trace"
dynamic = ["version"]
```